Dockerfile

```diff
@@ -15,35 +20,37 @@ COPY go.mod go.sum /kubeseal-web/
 RUN go mod download \
     && go get github.com/markbates/pkger/cmd/pkger
 
-
-FROM dev as build
+FROM --platform=$BUILDPLATFORM dev AS build
+ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
 
 COPY ./ /kubeseal-web/
 
-RUN mkdir /build/
-
-RUN pkger -include /static/ \
-    && rm -rf /kubeseal-web/static/
-
-RUN CGO_ENABLED=0 \
-    go build -v \
-    -ldflags "-s" -a -installsuffix cgo \
-    -o /build/kubeseal-web \
-    /kubeseal-web/ \
+RUN mkdir /build/ \
+    && pkger -include /static/ \
+    && rm -rf /kubeseal-web/static/ \
+    && CGO_ENABLED=0 GOOS=${TARGETOS} GOARCH=${TARGETARCH} go build -v \
+       -ldflags "-s -w" -a -installsuffix cgo \
+       -o /build/kubeseal-web . \
     && chmod +x /build/kubeseal-web
 
-
-FROM alpine:3.12 AS prod
+FROM --platform=$TARGETPLATFORM alpine:3.12 AS prod
+ARG TARGETPLATFORM
+ARG TARGETOS
+ARG TARGETARCH
 
 LABEL org.label-schema.vcs-url="https://github.com/MrSaints/kubeseal-web" \
       maintainer="Ian L. <os@fyianlai.com>"
 
 RUN apk add --no-cache bash ca-certificates curl jq wget nano
 
-COPY --from=build /build/kubeseal-web /kubeseal-web/run
+# Check required to handle naming convention in kubeseal assets as it doesnt contain the "linux-" prefix
```

Comment:
It seems it does in newer versions. Could we just upgrade? Any breaking changes?

Reply:
Will go ahead update to latest, seems like breaking changes are just Go versions but we're already upgrading it to 1.19